src/components/Button.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled from 'styled-components/macro';
```

Reply:
Unfortunately there is a bug with the latest `styled-components` version and `macro` does nothing 😿

Reply:
🙀